ruuvitag_sensor/data_formats.py

```diff
@@ -103,6 +104,9 @@ def convert_data(raw: str) -> DataFormatAndRawSensorData:  # noqa: PLR0911
         if candidate.startswith("FF990406"):
             return (6, candidate[6:])
 
+        if candidate.startswith("FF9904E1"):
+            return ("E1", candidate[6:])
```

Comment:
As this returns DataFormatAndRawSensorData, we need to change DataFormat  to support also str

https://github.com/ttu/ruuvitag-sensor/blob/62ee2f6c3d8a145e030c25462fb8547d89252cb3/ruuvitag_sensor/ruuvi_types.py#L65